packages/demo/webpack.config.js

```diff
@@ -11,7 +10,7 @@ const CopyPlugin = require('copy-webpack-plugin');
  */
 module.exports = {
     entry: {
-        main: './src/Index.tsx',
+        main: './built/Index.js',

```

Comment:
:thinking: Does this means that the build is done before webpack start instead for webpack doing the building? :eyes:

Reply:
Yes, TypeScripts compiler runs before webpack to convert .ts files to .js and outputs the result into `built` folder. Then webpack bundles everything together from those .js file inside `built` folder.

Reply:
Does this slow down the hot reload time a bit? :eyes: Since you have a two step build kinda :thinking: 
Or tsc is so fast anyway it's not an issue? :laughing:

Reply:
In theory it should indeed be slower, but in practice when I tried it I didn't notice any difference in speed 🤷

Reply:
Good to know :sweat_smile: :woman_shrugging: :rocket: